docker/package/fedora.py

```diff
@@ -59,13 +61,39 @@ def build_fedora_package(
         build_deps + run_deps, run_deps, f"{home}/rpmbuild/SPECS/{pkg.name}.spec"
     )
     os.rename(f"{dir}.tar.gz", f"{home}/rpmbuild/SOURCES/{dir}.tar.gz")
-    subprocess.run(
-        [
-            "rpmbuild",
-            "-bs" if is_source else "-bb",
-            f"{home}/rpmbuild/SPECS/{pkg.name}.spec",
-        ],
-        check=True,
-    )
+    for dist in fedora_versions:
+        if not is_source and dist == "native":
+            subprocess.run(
+                [
+                    "rpmbuild",
+                    "-bb",
+                    f"{home}/rpmbuild/SPECS/{pkg.name}.spec",
+                ],
+                check=True,
+            )
+        else:
+            subprocess.run(
+                ["rpmbuild", "-bs", f"{home}/rpmbuild/SPECS/{pkg.name}.spec"],
+                check=True,
+            )
+            if not is_source:
+                os.makedirs(f"{home}/rpmbuild/RPMS/x86_64", exist_ok=True)
+                subprocess.run(
+                    [
+                        "mock",
+                        "--resultdir",
+                        f"{home}/rpmbuild/RPMS/x86_64",
+                        "-r",
+                        f"/etc/mock/fedora-{dist}-x86_64.cfg",
```

Comment:
Do these configs always exist?

Reply:
Yes, it's stable paths if you fix host system version